pkg/app/app.go

```diff
@@ -164,9 +165,36 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
+	} else {
+		ticker := time.NewTicker(interval)
```

Comment:
I have no clue what our minimal interval is. I did do testing as low as 2s, which caused craziness in the logs since the broker was being peppered with requests. I'm sure if someone sets it that low it will be clear what wrong since  ```a.log.Debug("RefreshInterval: %v", interval.String())``` will be constantly showing up in the logs.

I think as far as we should go is warning that the value is low. But even if we did that, it's still pretty obvious what's happening because the logs will be flooded with TONS of spec data plus the interval message.